### DISCUSSION TITLE
Does it have codebase awareness?

### DISCUSSION THREAD

**Discussion by zeroaddresss** [CopilotC-Nvim/CopilotChat.nvim#399] [Q&A] [answered]
When evaluating user code, is it considering the whole codebase or is it limited to current buffer / currently selected block?

**Comment by jellydn** [comment]
We have this partially base on `buffer` or `buffers` depends on the usage. 

cc @gptlang @deathbeam

**Comment by deathbeam** [comment, marked as answer]
#files is basically what you want
